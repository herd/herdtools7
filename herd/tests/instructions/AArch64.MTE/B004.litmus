AArch64 B004
Variant=async
{
 0:X1=x:red
}
 P0          ;
 MOV W0,#1   ;
 STR W0,[X1] ;
forall([x]=1 /\ 0:TFSR_ELx=1)
