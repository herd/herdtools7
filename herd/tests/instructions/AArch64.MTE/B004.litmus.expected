Test B004 Required
States 1
0:TFSR_ELx=1; [x]=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([x]=1 /\ 0:TFSR_ELx=1)
Observation B004 Always 1 0
Hash=a8708b099b5f95ad916f70e33c905ffc

